packages/block-library/src/embed/embed-placeholder.native.js

```diff
@@ -39,7 +40,7 @@ const EmbedPlaceholder = ( {
 		[ clientId ]
 	);
 	const [ isEmbedSheetVisible, setIsEmbedSheetVisible ] = useState(
-		isSelected && wasBlockJustInserted && ! value
+		isSelected && ( ( wasBlockJustInserted && ! value ) || isEditingURL )
```

Comment:
This change enables the bottom sheet to be automatically displayed when the replace action is triggered.